app/src/main/java/com/framgia/music_44/screen/home/adapter/HomeAdapter.java

```diff
@@ -0,0 +1,73 @@
+package com.framgia.music_44.screen.home.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.bumptech.glide.Glide;
+import com.framgia.music_44.R;
+import com.framgia.music_44.data.model.Songs;
+import com.framgia.music_44.util.ItemRecyclerViewClickListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeAdapter extends RecyclerView.Adapter<HomeAdapter.ViewHolder> {
+    private List<Songs> mSongsList;
+    private Context mContext;
+    private ItemRecyclerViewClickListener mListener;
+
+    public HomeAdapter(Context context, ItemRecyclerViewClickListener listener) {
+        mContext = context;
+        mListener = listener;
+        mSongsList = new ArrayList<>();
+    }
+
+    public void updateList(List<Songs> songsList) {
+        mSongsList.addAll(songsList);
+        notifyDataSetChanged();
+    }
+
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        LayoutInflater layoutInflater = LayoutInflater.from(viewGroup.getContext());
+        View view = layoutInflater.inflate(R.layout.item_list, viewGroup, false);
+        return new ViewHolder(view, mListener);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
+        Glide.with(mContext).load(mSongsList.get(i).getImage()).into(viewHolder.mImageViewSong);
```

Comment:
tạo 1 method riêng xử lý phần bindviewData
```suggestion
        viewHolder.bindDataView(mSongsList.get(i))
```